scripts/run_build.sh

```diff
@@ -0,0 +1,2 @@
+docker build --target production --tag sm4o/todo.app:latest --tag sm4o/todo.app:$TRAVIS_COMMIT .
+docker push sm4o/todo.app --all-tags
```

Comment:
Worth noting that you're pushing to your latest branch on every commit, not just on the master branch; usually we'd reserve that for the latest _stable_ version, so e.g. include these commands in your deploy script rather than every time we run the tests successfully

Reply:
Ack. I'm adding it on the module_9 (#10) branch